spec/classes/letsencrypt_install_spec.rb

```diff
@@ -21,6 +22,14 @@
       let :facts do
         facts
       end
+      let(:vcs_dependencies) do
+        case facts[:osfamily]
+        when 'Debian'
+          ['git', 'python3', 'python3-venv']
+        when 'RedHat', 'OpenBSD', 'FreeBSD'
```

Comment:
I don't see data files for the BSDs to set there properties. How are these set?

Reply:
I see that there are data/FreeBSD-family.yaml and data/OpenBSD-family.yaml files, but that they do not have letsencrypt::vcs_dependencies sections in them.  I do not know what the dependencies for letsencrypt would be on those systems to try adding them.  Is that what is blocking this PR?